docs/rails.md

```diff
@@ -0,0 +1,35 @@
+# Rails conventions
+
+## App behavior should be configured via environment variables
+
+When implementing new features, it should not depend on current
+`RAILS_ENV` value, but environment variable(s) specific to the
+feature.
+
+* Avoid modifying `config/environments/*.rb`, and keep default
+  rails configuration (this will also help when updating rails);
+* Avoid testing:
+  + `Rails.env.development?`;
```

Comment:
Can we keep `*` rather than `+`? Changing this bullet is redundent with the indentation level.

Reply:
Also, not a big deal, but I would either use dots at the end of each item, or nothing at all, but not different trailing characters.

Reply:
> Can we keep * rather than +? Changing this bullet is redundent with the indentation level.

Done.

> Also, not a big deal, but I would either use dots at the end of each item, or nothing at all, but not different trailing characters.

I chose to remove last char (`;` or `.`) on each line. Using `.` on other items
than the last one looks strange for me, and I'm not sure if it's correct for an
enumeration.